File: 2st/Yeonwoo/수 조작하기 2.py
def solution(numLog):
    answer = ''
    for i in range(len(numLog)-1):
        if numLog[i+1] == numLog[i]+1:
            answer += 'w'
        elif numLog[i+1] == numLog[i]-1:
            answer += 's'
        elif numLog[i+1] == numLog[i]+10:
            answer += 'd'
        elif numLog[i+1] == numLog[i]-10:
            answer += 'a'
    return answer

File: 4st/Gau/programmer/수 조작하기 2.java
class Solution {
    public String solution(int[] numLog) {
        
        String answer = "";
        
        for (int i = 0; i < numLog.length - 1; i++) {
            int j = numLog[i + 1] - numLog[i];
            
            if (j == 1) {
                answer += "w";
            } else if (j == -1) {
                answer += "s";
            } else if (j == 10) {
                answer += "d";
            } else if (j == -10) {
                answer += "a";
            }
        }
        
        return answer;
    }
}
